### PR TITLE
Add CO2 filtering for Cryo

### PR DESCRIPTION
**Pull request by timothymtorres**
## About The Pull Request

This adds a CO2 filter and hooks the filter to the waste loop.

StrongDMM_DFkIjn7I45

StrongDMM_GsLt3jbYDe

StrongDMM_hh8JO2PxG3

StrongDMM_YMDUOFN3ao

StrongDMM_0FEkG1U0SR

## Why It's Good For The Game

A lot of people were complaining about this in coderbus.  This is a simple mapping fix.

## Changelog

:cl:
fix: Fix Cryo areas to have CO2 filtering from the gas loops.
/:cl:

### CODE REVIEW

**Comment by CRITAWAKETS** [comment]
Why doesn't it just put the CO2 in waste?

**Comment by timothymtorres** [comment]
There are 3 options:

### Swap out the passive vent for a connector port with an empty CO2 canister on top

Pros: CO2 waste should never get cloggged since filling a canister with breathed CO2 would take hours
Cons: Less room to maneuver 

### Leave the passive vent

Pros: CO2 waste gets dumped into the room.  The scrubbers in the room collect the CO2 and suck it into the waste loop.
Cons: The CO2 waste that gets vented into the room is very cold.  There is such a tiny amount that this shouldn't messup the room temperature though.

### Remove passive vent and hook directly into waste loop

Pros: Less likely for someone to accidentally (or intentionally) leak O2 or CO2 into the room by messing up the filter
Cons: If the waste loop is clogged from a plasma fire or someone dumps the SM excess gas into waste then the cryo filter can't pump gas and it stays there.

Given these different solutions, I think the passive vent is the best option.

**Comment by Ryll-Ryll** [comment]
I'd reword this to say Adds CO2 filtering rather than fixes, since the CO2 buildup was intended by the original PR and it said as much. That said, I'll still keep the tags as is

**Comment by LemonInTheDark** [comment]
You have a line of the map that is commented out, in the icebox dmm. you will need to fix that before a merge

**Comment by timothymtorres** [comment]
> You have a line of the map that is commented out, in the icebox dmm. you will need to fix that before a merge

Fixed with https://github.com/tgstation/tgstation/pull/65676/commits/55d3bb8f71c5e9cbf062075ecc33f523a1e43ee5
